Hunting Queries/MultipleDataSources/SQLAlertCorrelationwithCommonSecurityLogsandAuditLogs.yaml

```diff
@@ -0,0 +1,52 @@
+id: dc5adcc9-70ab-4fba-8690-f57767e8ca02
+name: SQL Alert Correlation with CommonSecurityLogs and AuditLogs
+description: |
+  'This query combines different SQL alerts with CommonSecurityLogs and AuditLogs helping analysts /investigate any possible SQL related attacks faster
+   thus reducing MTTR'
+requiredDataConnectors:
+  - connectorId: PaloAltoNetworks
+    dataTypes:
+      - CommonSecurityLog
+  - connectorId: AzureSecurityCenter
+    dataTypes:
+      - SecurityAlert (ASC)
```

Comment:
I think this should be just 'SecurityAlert'

Reply:
Looks like this is the valid data type for ASC alerts  as per Wiki -https://github.com/Azure/Azure-Sentinel/wiki/Query-Style-Guide#datatypes